src/Domain/Creators/Cylinder.cpp

```diff
@@ -34,104 +24,113 @@ Cylinder::Cylinder(
     typename IsPeriodicInZ::type is_periodic_in_z,
     typename InitialRefinement::type initial_refinement,
     typename InitialGridPoints::type initial_number_of_grid_points,
-    typename UseEquiangularMap::type use_equiangular_map) noexcept
+    typename UseEquiangularMap::type use_equiangular_map,
+    typename RadialPartitioning::type radial_partitioning,
+    typename HeightPartitioning::type height_partitioning,
+    std::unique_ptr<domain::creators::time_dependence::TimeDependence<3>>
```

Comment:
I think you're missing the includes `Domain/Creators/TimeDependence/TimeDependence.hpp`, `Domain/Creators/TimeDependence/None.hpp` and `<memory>`

Also add `<vector>`, `<array>`, `unordered_map`